### PR TITLE
config: enable cherrypicker for ticdc

### PR DESCRIPTION
**Pull request by hi-rustin**
part of https://github.com/ti-community-infra/configs/issues/278

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

Waiting for @amyangfei approve

Waiting for @amyangfei modify labels

Waiting for stop sre-bot service

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -588,9 +588,9 @@ ti-community-label:
       - 'good-first-issue'
       - 'hacktoberfest'
       - 'internal-test'
-      - 'needs-cherry-pick-4.0'
-      - 'needs-cherry-pick-5.0'
-      - 'needs-cherry-pick-5.0-rc'
+      - 'needs-cherry-pick-release-4.0'
+      - 'needs-cherry-pick-release-5.0'
+      - 'needs-cherry-pick-release-5.0-rc'
```
Comment (amyangfei) [inline prow/config/external_plugins_config.yaml:593]: We can remove this label now, branch `release-5.0-rc` is not maintained any more

**Review by amyangfei — APPROVED** [review]

File: prow/config/external_plugins_config.yaml (hunk 1 above)
Comment (hi-rustin) [reply prow/config/external_plugins_config.yaml:593]: Got it. I will remove it.

**Pull request merged**